src/js/events/MesosStateActions.js

```diff
@@ -37,8 +51,31 @@ var MesosStateActions = {
       };
     },
     {delayAfterCount: Config.delayAfterErrorCount}
-  )
+  ),
 
+  fetchNodeState: RequestUtil.debounceOnError(
+    Config.getRefreshRate(),
+    function (resolve, reject) {
+      return function (task, node, success, error) {
+        return RequestUtil.json({
+          url: getNodeStateURL(task, node),
+          success(response) {
+            resolve();
+            success(response);
```

Comment:
We could. I was just moving code around at this point. It didn't make sense to me that it was listed where it was.